pkg/storage/filesystem.go

```diff
@@ -20,24 +20,25 @@ type Mountpoint struct {
 	FsType     string
 }
 
+func GetSupportedFilesystems() (fstypes []string, err error) {
+	return internalGetFilesystems("/proc/filesystems")
+}
+
 // GetSupportedFilesystems returns the supported file systems for block devices,
```

Comment:
move the comment

Reply:
Should I keep the comment the same, but move it up to GetSupportedFilesystems() ??

Reply:
yes, thanks!